test/onnx/test_fx_to_onnx.py

```diff
@@ -677,6 +677,35 @@ def forward(self, input: torch.Tensor):
 
         _ = torch.onnx.dynamo_export(Float8Module(), torch.randn(1, 2, 3, 4))
 
+    def test_checkpoint_cast(self):
+        model_id = "openai/whisper-large-v3"
```

Comment:
nit: better add this model to onnx ci model cache https://github.com/pytorch/pytorch/blob/7718a1cd4f8e0b794c18a31ebd6353d6273c534e/.ci/docker/common/install_onnx.sh#L38